package.json

```diff
@@ -2,17 +2,36 @@
   "name": "personal-site",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:5000",
   "dependencies": {
+    "@apollo/client": "^3.6.9",
+    "@emotion/react": "^11.10.0",
+    "@emotion/styled": "^11.10.0",
+    "@mui/icons-material": "^5.8.4",
+    "@mui/material": "^5.10.1",
+    "@reduxjs/toolkit": "^1.8.5",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
+    "body-parser": "^1.20.0",
+    "express": "^4.18.1",
+    "graphql": "^16.6.0",
+    "nodemon": "^2.0.19",
+    "pg": "^8.8.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.0.2",
```

Comment:
I don't think this project is probably complicated or deep enough to justify Redux.  Dan Abramov, one of the people who created Redux has stated that most React projects don't actually need Redux.  This is one of the first paragraphs in the Redux FAQ.  
![image](https://user-images.githubusercontent.com/25065036/189394062-c6f657c9-0b4d-44d7-a722-ba537e911eb1.png)\
Similar to what i was saying about DBs, if you want to use this tech, start a more complicated project.